docs/user-guide/ze-ftp-using-ze-ftp-ext.md

```diff
@@ -0,0 +1,45 @@
+# Using the FTP Extension
+
+## Prerequisites
+
+Ensure that you obtain remote access to z/OS FTP service before you can use the extension.
+
+1. Connect to z/OS with the FTP client.
+1. Run the following command in the FTP client:
+
+   ```bash
+   rstat
+   ```
+
+1. Confirm that the `JESINTERFACELevel` option is set to `2`.
+
+## Using
+
+To use the FTP Extension with Zowe Explorer:
+
+1. Select the **Zowe Explorer** icon on the **Activity Bar** in VS Code.
+
+2. Hover over the **DATA SETS**, **UNIX SYSTEM SERVICES (USS)**, or **JOBS** bar and select the corresponding **+** icon to view the Zowe CLI FTP profiles in the **picker** dropdown list.
+
+    If you do not have an existing FTP profile, see [Creating an FTP profile with Zowe Explorer](../user-guide/ze-ftp-using-ze-ftp-ext#creating-an-ftp-profile-with-zowe-explorer).
+
+3. Select a profile to display it in the **Side Bar**.
+
+4. Hover over the profile and click the **Search** icon.
+
+5. Enter the applicable values in the **picker** field:
+    - For data sets, select or enter the data set name.
+    - For USS, select or enter the path.
+    - For jobs, select or enter the job owner and job prefix.
+
+## Creating an FTP profile with Zowe Explorer
```

Comment:
@zFernand0 with create profile actions should we be promoting use of v2?

Reply:
I agree we should 😋 
But maybe it require it's own section in `Using Zowe Explorer` that the FTP section can link to (and possibly expand)
I believe this is just content that was there from before 😋

Reply:
@zFernand0 @JillieBeanSim  I agree to work on this on a separate PR